openverse_catalog/dags/util/loader/paths.py

```diff
@@ -148,3 +159,22 @@ def _extract_media_type(tsv_file_name: Optional[str]) -> str:
     except (AttributeError, IndexError):
         media_type = "image"
     return media_type
+
+
+def _get_tsv_version(tsv_file_name: str) -> str:
+    """TSV file version can be deducted from the filename
+    v0: without _vN_ in the filename
+    v1+: has a _vN in the filename
+
+    >>>_get_tsv_version('/behance_image_20210906130355.tsv')
+    '000'
+    >>> _get_tsv_version('/jamendo_audio_v005_20210906130355.tsv')
+    '005'
+    """
+
+    version_pattern = re.compile(r"_v(\d+)_")
```

Comment:
Since we're already using 3 characters for this, should we perhaps use:
```suggestion
    version_pattern = re.compile(r"_v(\d{3})_")
```
This would ensure we're only matching cases with exactly 3 digits after the `v`, just ince case the source name happens to be of that form as well? I might be being a bit too paranoid :slightly_smiling_face:

Reply:
I wasn't sure I had enough zeros in the version, so I left it like that. I guess if we all agree that 3-digit version number is good enough, I will update the regex.

Reply:
I think 3 digits is good - I would be surprised if we made 1000 updates (famous last words) to the TSV file versions :grin: